### ISSUE TITLE
Automatic removal of resting staves

### ISSUE THREAD

**Issue by craigsapp** [polyrhythm-project/polyrhythm-website#54]
There is a rule to remove resting parts (in the RDS scores during the processing of the grouping information).

T698 has a problem:

<img width="902" alt="Screen Shot 2020-01-22 at 01 17 24" src="https://user-images.githubusercontent.com/3487289/72881223-338f5500-3cb5-11ea-9c2b-60a3d10953a4.png">

gets converted to:

<img width="858" alt="Screen Shot 2020-01-22 at 01 17 29" src="https://user-images.githubusercontent.com/3487289/72881237-3b4ef980-3cb5-11ea-9c84-d5cfbbf063d6.png">

It would be best to not remove piano part resting staves (i.e., a grand-staff resting staff).  This will probably mean adding some sort of feature to the extract filter.

**Comment by craigsapp** [comment]
See issue https://github.com/polyrhythm-project/polyrhythm-website/issues/63 where problem also occurs in T337

**Comment by Boardbreak** [comment]
ex. T444, Berg, missing bass clef of its grand staff, as well as its instrument name.

VHV:

![Celesta - VHV](https://user-images.githubusercontent.com/59404234/73564533-a4044780-4414-11ea-9a80-4e6978036a94.png)

Desired:

![Celesta - Desired ](https://user-images.githubusercontent.com/59404234/73564547-aa92bf00-4414-11ea-910d-2e02335b8120.png)

**Comment by epoudrier** [comment]
I confirm Craig's note that we decided to eliminate resting staves. Preserving both staves of the piano if only one is resting does seem like a good idea from a visual point of view. We will need to take this into account when we do analysis where number of parts are counted (i.e., whether we want to remove single resting piano staves from the equation.

**Comment by craigsapp** [comment]
Yes, I made a fix to exclude removal of staves that are all rests provided that they belong to a part that has another staff that does not have all rests, but I got distracted from reprocessing the data with the fixed extraction feature.
